update.php

```diff
@@ -4,45 +4,82 @@
     <title>
         
     </title>
+    <link rel="stylesheet" type="text/css" href="styles/main.css">
 </head>
 <body>
     <?php
-if(isset($_POST['update']))
-{
-    
-    session_start();
-    $name = $_POST['name'];
-    $friend = $_POST['friend'];
-    $isbn = $_POST['isbn'];
-    
-    $servername = "localhost";
-    $username = $_SESSION['username'];
-    $password = $_SESSION['password'];
-    $dbname = "books_lending";
-    $table = "books";
-
-    $conn = new mysqli($servername, $username, $password, $dbname);
-    $sql = "INSERT INTO $table (id, name, isbn, friend)
-    VALUES (null, '$name', '$isbn', '$friend')";
-    if ($conn->query($sql) === TRUE) {
-        echo "New record created successfully <a href='view.php'>return to table</a><br>";
-    } else {
-        echo "Error: " . $sql . "<br>" . $conn->error;
-    }
-    
 
-}else
-{
-    //echo "ddd";
-}
+        if(isset($_POST['update'])) {
```

Comment:
better modify than delete and modify. use git instead of website ^^

Reply:
I use GIT but i had to delete and modify that file for a merge conflict, i tried to use "git mergetool" but didn't get results as i expected.

Now it should be fine, i connected my account in GIT to the forked repo and as far as i can see, it is adding automatically any change i make directly in my Pull Request (this might seems banal, but initially i thought that any change should have a specific Pull Request for each file, nice to see i was wrong about that).

Reply:
You have branches for grouping changes. Nice CSS on login form !